fanuc_lrmate200id_moveit_config/package.xml

```diff
@@ -29,6 +29,7 @@
 
   <buildtool_depend>catkin</buildtool_depend>
 
+  <exec_depend>fanuc_lrmate200id_gazebo</exec_depend>
```

Comment:
The problem with adding this `exec_depend` is that all users -- even those who just want to use `fanuc_lrmate200id_moveit_config` with real hardware -- will have to install all of Gazebo, and all its dependencies. That doesn't seem very nice - especially not when you want to run `move_group` on a headless machine.

In the case of the ABB IRB 120(T) and its Gazebo support package, for that reason, we chose to keep the Gazebo dependency (brought in transitively through `abb_irb120_gazebo`) optional, by not including it in `abb_irb120_moveit_config/package.xml`.

Instead, we only included the `moveit_planning_execution_gazebo.launch` file, and added an explanatory piece of text at the top (see [abb_irb120_moveit_config/launch/moveit_planning_execution_gazebo.launch](https://github.com/ros-industrial/abb_experimental/blob/91de854692a8166014f8ec8e3b9565299b79c27d/abb_irb120_moveit_config/launch/moveit_planning_execution_gazebo.launch)).

I'd prefer something like that for this `lrmate200id` package as well, and would appreciate it if you could make the necessary changes. It'd also mean renaming the `gazebo_demo.launch` to `moveit_planning_execution_gazebo.launch`.